### PR TITLE
feat: a definition for vcard:kind is added

### PR DESCRIPTION
**Pull request by xiaofengleo**
a definition for vcard:kind is added in Dataset.ttl

### CODE REVIEW

File: Formalisation(shacl)/Core/PiecesShape/Dataset.ttl [hunk 1]
```diff
@@ -153,3 +154,26 @@
   ];
 
 sh:targetClass foaf:Agent .
+
+
+:KindShape a sh:NodeShape;
+  sh:property     
+  [ 
+    sh:path vcard:hasEmail ;
+    sh:nodeKind sh:IRI ;
```
Comment (kburger) [inline Formalisation(shacl)/Core/PiecesShape/Dataset.ttl:163]: As discussed with Patrick, maybe we can add a `sh:pattern "mailto:\w+@\w+\.\w+"` here.

File: Formalisation(shacl)/Core/PiecesShape/Dataset.ttl [hunk 2]
```diff
@@ -153,3 +154,26 @@
...
+    dash:viewer dash:LabelViewer ;
+    dash:editor dash:URIEditor ;
```
Comment (kburger) [inline Formalisation(shacl)/Core/PiecesShape/Dataset.ttl:165]: Not sure if `dash:LabelViewer` is the most appropriate here, there will not be a label to resolve for this.

File: Formalisation(shacl)/Core/PiecesShape/Dataset.ttl (hunk 1 above)
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/PiecesShape/Dataset.ttl:163]: done

File: Formalisation(shacl)/Core/PiecesShape/Dataset.ttl (hunk 2 above)
Comment (xiaofengleo) [reply Formalisation(shacl)/Core/PiecesShape/Dataset.ttl:165]: changed to dash:viewer dash:URIViewer.
There are other dash:LabelViewer pair with dash:URIEditor, which are working correctly so far. Do we need to change all to dash:URIViewer, or wait and do later?

File: Formalisation(shacl)/Core/PiecesShape/Dataset.ttl [hunk 3]
```diff
@@ -153,3 +154,27 @@
...
 sh:targetClass foaf:Agent .
+
+
+:KindShape a sh:NodeShape;
+  sh:property     
+  [ 
+    sh:path vcard:hasEmail ;
+    sh:nodeKind sh:IRI ;
+    sh:pattern "mailto:\w+@\w+\.\w+"
+    dash:viewer dash:URIViewer
```
Comment (kburger) [inline Formalisation(shacl)/Core/PiecesShape/Dataset.ttl:165]:
```suggestion
    sh:pattern "mailto:\\w+@\\w+\\.\\w+" ;
    dash:viewer dash:URIViewer ;
```

**Review by kburger — CHANGES_REQUESTED** [review]

**Review by xiaofengleo — COMMENTED** [review]
looks good.

**Review by xiaofengleo — COMMENTED** [review]
looks good

**Review by kburger — APPROVED** [review]
The regex might be improved, but good enough for now.

**Pull request merged**